hack/observability/visualizer/values.yaml

```diff
@@ -3,9 +3,8 @@ replicas: 1
 image:
   repository: ghcr.io/jont828
   name: cluster-api-visualizer
+  tag: latest # TODO: do we want to pin this to latest?
```

Comment:
Q: I assume your image is hosted on the GitHub image registry? Is it now possible to overwrite a tag?

I'm asking because 1-2 years ago that was only possible by opening a ticket, because GitHub didn't like folks just moving docker image tags.

Reply:
Yep the image is hosted on GitHub container registry. I just set it up so that the default image tag is pulled from `Chart.appVersion` and if `Values.image.tag` is set it will overwrite it. I'm not sure if that counts as moving image tags.

Reply:
What I mean is that previously you were not able to overwrite a tag in the image registry.

This means if you push a latest tag once, you can't overwrite it later by pushing another image under latest again.

Essentially all image tags were immutable.

Reply:
As far as I know I've been able to rebuild the v1.0.0 image and push to GHCR without any issues.

Reply:
Okay thx!